File: Conditionals/oldEnough.py
age = int(input("how old are you?\n"))
if age >= 18:
    print("you are old enough to vote!")
elif age <18:
    print("you are not old enough to vote:(")
elif age <16:
    print("you are not old enough to drive:(")
elif age <15:
    print("you are not old enough to get your learners permit:(")
elif age <5:
    print("you are not old enough to go to school:(")
else:
    print("you are too young to go to school")

File: Conditionals/oldEnough.c
#include <stdio.h>

int age;

int main (void){
    printf("how old are you?\n");
    scanf("%d", &age);
    if (age >= 18){
        printf("you are old enough to vote!\n");
    }else if(age >= 16){
        printf("you are old enough to drive\n");
    }else if(age >= 16){
        printf("you are old to get learners permit\n");
    }else if(age >= 5){
        printf("you are old enough to go to school\n");
    }
    return 0;
}